src/Symfony/Component/Form/Extension/Validator/EventListener/ValidationListener.php

```diff
@@ -35,10 +35,6 @@ public static function getSubscribedEvents()
         return array(FormEvents::POST_SUBMIT => 'validateForm');
     }
 
-    /**
-     * @param ValidatorInterface       $validator
-     * @param ViolationMapperInterface $violationMapper
-     */
```

Comment:
they provide no value at all, this is just an exact repetition of the method signature